appservice/src/main/resources/static/swagger/swagger-spec.yaml

```diff
@@ -1631,20 +1647,26 @@ definitions:
         allOf:
           - $ref: '#/definitions/Curies'
           - $ref: '#/definitions/SelfLink'
+      date:
+        type: string
+        example: 2016-03-16
+        description: The day date in ISO 8601 format (yyyy-MM-dd)
       _embedded:
         properties:
           "yona:dayActivities":
             type: array
             items:
               $ref: '#/definitions/DayActivity'
+    required:
+      - date
   DayActivity:
     allOf:
       - $ref: '#/definitions/IntervalActivity'
     properties:
       date:
         type: string
         example: 2016-03-16
-        description: The day date in ISO 8601 format (yyyy-MM-dd)
+        description: The day date in ISO 8601 format (yyyy-MM-dd). Only included in day detail
```

Comment:
Same answer: 
- In day overview, every `DayActivity` has the same date as its parent `DayActivityOverview`, so it is not necessary to include here. 
- In week activity/week activity overview, I could add the date if you like. However in the original design the date was not displayed, only the day of the week. There is a self link already to get the date if necessary, so I would propose to not include it.